### PR TITLE
SEPM enhancement

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready (except unit testing)

## Related Issues
fixes: https://github.com/demisto/etc/issues/18592

## Description
Adding a few new outputs.
Creating a new command (after talking to Arseny)
Modifying Outputs to match the new Indicator format

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by orlichter1** [comment]
@kirbles19 Could you please go over the [release notes](https://github.com/demisto/content/edit/Sep-enhancement/Integrations/SymantecEndpointProtection_V2/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F4178) and the [yml](https://github.com/demisto/content/edit/Sep-enhancement/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F4178)?

**Comment by kirbles19** [comment]
@ygerowitz I reviewed the .md, just review the .yml and be in touch with @orlichter1 with any questions.

File: Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py [hunk 1]
```diff
@@ -356,30 +402,43 @@ def get_endpoints_info(token, computer_name, last_update, os, page_size, columns
...
+        group = content.get('group', {'name': ''})
+        bool_start = group.get('name').startswith(group_name[:-1]) if group_name and group_name[-1] == '*' else False
+        if (not group_name) or group.get('name') == group_name or bool_start:  # No group name filter
+            # used `set` on the mac address list as it sometimes contained duplicated values
+            content['macAddresses'] = list(set(content.get('macAddresses')))
+            entry_context.append({
+                'Hostname': content.get('computerName'),
+                'Domain': content.get('domainOrWorkgroup'),
+                'IPAddresses': content.get('ipAddresses'),
+                'OS': content.get('operatingSystem') + ' | ' + content.get('osBitness'),
```
Comment (yaakovi) [inline Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py]:
```suggestion
                'OS': content.get('operatingSystem', '') + ' | ' + content.get('osBitness', ''),
```

File: Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py [hunk 2]
```diff
@@ -481,6 +552,29 @@ def system_info_command(token):
     })
 
 
+def old_clients_command(token):
+
```
Comment (yaakovi) [inline Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py]: remove empty line

File: Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py [hunk 3]
```diff
@@ -481,6 +552,29 @@ def system_info_command(token):
...
+    columns = demisto.getArg('columns')
+    group_name = demisto.getArg('groupName')
+    desired_version = demisto.getArg('desiredVersion')
+    filtered_json_response, entry_context = get_endpoints_info(token, computer_name, last_update, os, page_size,
+                                                               columns, group_name)
+    columns_list = choose_columns(columns, ENDPOINTS_INFO_DEFAULT_COLUMNS)
+    filtered_json_response = filter_only_old_clients(filtered_json_response, desired_version)
+    md = create_endpints_filter_string(computer_name, last_update, os, page_size, group_name)
+    md += tableToMarkdown('Old Endpoints', filtered_json_response, columns_list)
+    demisto.results({
```
Comment (yaakovi) [inline Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py]: add empty line here

File: TestPlaybooks/playbook-SymantecEndpointProtection_Test.yml [hunk 4]
```diff
@@ -1230,19 +1267,56 @@ tasks:
...
+    task:
+      id: 74af733e-4356-4f9f-8589-4ee1e75f0cdd
+      version: -1
+      name: sep-identify-old-clients
+      description: Get endpoints with a running version that isnt the same as the
+        target version or isn't consistent with the desired version (if specified).
+      script: Symantec Endpoint Protection V2|||sep-identify-old-clients
+      type: regular
+      iscommand: true
+      brand: Symantec Endpoint Protection V2
```
[the remark this answers is not in the document]
Comment (orlichter1) [reply TestPlaybooks/playbook-SymantecEndpointProtection_Test.yml]: What do you mean? It was generated automatically from 
image
should I just write "" instead? would that work?

[the remark this answers is not in the document]
Comment (yaakovi) [reply TestPlaybooks/playbook-SymantecEndpointProtection_Test.yml]: when choosing the task automation, choose the one with out the brand in brackets.
for example with gmail (need to choose the first):
image

**Comment by orlichter1** [comment]
@yaakovi changed the things you requested.
@ygerowitz, @kirbles19 - The last update thing we talked about is actually an issue in Demisto that I opened a bug for. Thanks, guys :) I've added the lastUpdate and 2 more context fields could you have a look at those additions as well.

**Comment by ygerowitz** [comment]
Updated Symantec_Endpoint_Protection_V2 yml file.

File: Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py [hunk 5]
```diff
@@ -425,7 +425,8 @@ def get_endpoints_info(token, computer_name, last_update, os, page_size, columns
 
 def create_endpints_filter_string(computer_name, last_update, os, page_size, group_name=None):
     md = '## Endpoints Information'
-    md += ', filtered for last updated status: {}'.format(last_update) if last_update else ''
+    if last_update != '0':
+        md += ', filtered for last updated status: {}'.format(last_update) if last_update else ''
```
Comment (yaakovi) [inline Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py]: it's weird that you start with a comma

File: Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py [hunk 6]
```diff
@@ -425,7 +425,8 @@ def get_endpoints_info(token, computer_name, last_update, os, page_size, columns
...
     md += ', filtered for hostname: {}'.format(computer_name) if computer_name else ''
```
Comment (yaakovi) [inline Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py]: this is also a possible start...

**Comment by ygerowitz** [comment]
The Symantec documentation has been updated and will be published today. Here is the URL: https://support.demisto.com/hc/en-us/articles/360018859814

**Pull request merged**